aws/logs_monitoring/steps/enrichment.py

```diff
@@ -32,6 +33,12 @@ def enrich(events, cache_layer):
         extract_host_from_guardduty(event)
         extract_host_from_route53(event)
 
+        # Set tracing behavior for all step functions
+        if DD_STEP_FUNCTION_TRACE_ENABLED:
+            event[DD_CUSTOM_TAGS] = ",".join(
```

Comment:
Null check for `event[DD_CUSTOM_TAGS]` ?

Reply:
Will be moving this logic as per Kimi's comment and adding a null check on the entry